include/spdk/lvol.h

```diff
@@ -73,6 +73,7 @@ struct spdk_lvs_opts {
 	uint32_t		cluster_sz;
 	enum lvs_clear_method	clear_method;
 	char			name[SPDK_LVS_NAME_MAX];
+	const char *uuid;
```

Comment:
I think formatting should be:
```C
char		name[SPDK_LVS_NAME_MAX];
const char      *uuid;
```
?
Is the linter running?

Reply:
I works but missed this I am not sure why